plugin/src/main/groovy/com/cookpad/android/licensetools/LicenseToolsPlugin.groovy

```diff
@@ -72,6 +62,19 @@ class LicenseToolsPlugin implements Plugin<Project> {
             description = 'Check whether dependency licenses are listed in licenses.yml'
         }
 
+        def updateLicenses = project.task('updateLicenses').doLast {
+            initialize(project)
+
+            def notDocumented = dependencyLicenses.notListedIn(librariesYaml)
+            LicenseToolsExtension ext = project.extensions.findByType(LicenseToolsExtension)
+
+            notDocumented.each { libraryInfo ->
+                def text = generateLibraryInfoText(libraryInfo)
+                project.file(ext.licensesYaml).append("\n${text}")
+            }
+        }
+        updateLicenses.dependsOn('checkLicenses')
```

Comment:
`updateLicenses` might not be execute if `checkLicenses` failed.
so please remove this line. it should be declared on each projects.